File: Kattis/isyavowel/isyavowel.cpp
#include <iostream>
#include <string>

int main() {
    std::string compString;
    std::cin >> compString;
    std::string vowels = "aeiou";
    int count1 = 0, count2 = 0;
    int compString_length = compString.length();

    for (int i = 0; i < compString_length; i++) {
        for (int j = 0; j < 5; j++) {
            if (compString[i] == vowels[j]) {
                count1++;
                count2++;
            }
        }
        if (compString[i] == 'y') {
            count2++;
        }
    }
    std::cout << count1 << " " << count2 << "\n";
    return 0;
}

File: Kattis/isyavowel/isyavowel.py
compString = input()
vowels = "aeiou"

count1 = 0
count2 = 0

for char in compString:
    for vowel in vowels:
        if char == vowel:
            count1 += 1
            count2 += 1
    if char == 'y':
        count2 += 1

print(f"{count1} {count2}")